Droidcon-Boston/app/src/test/java/com/mentalmachines/droidcon_boston/preprocess/ConferenceDataModels.kt

```diff
@@ -0,0 +1,64 @@
+package com.mentalmachines.droidcon_boston.preprocess
```

Comment:
Retaining the Firebase keys (e.g. the speaker record "-L3dd0queRj8ffFi-_sx") might be helpful to get any specific lastUpdated Firebase results into the in-memory data objects quickly. Those keys might map into other Firebase tables later on, e.g. users publishing profiles alongside speakers.